src/status_im/constants.cljs

```diff
@@ -414,16 +414,19 @@
 (def ^:const ens-action-type-set-pub-key 2)
 
 ;; wallet
-(def ^:const ethereum-chain-id 1)
-(def ^:const goerli-chain-id 5)
-(def ^:const arbitrum-chain-id 42161)
-(def ^:const arbitrum-testnet-chain-id 421613)
-(def ^:const optimism-chain-id 10)
-(def ^:const optimism-testnet-chain-id 420)
+(def ^:const ethereum-mainnet-chain-id 1)
+(def ^:const ethereum-goerli-chain-id 5)
 (def ^:const ethereum-sepolia-chain-id 11155111)
+(def ^:const arbitrum-mainnet-chain-id 42161)
+(def ^:const arbitrum-goerli-chain-id 421613)
 (def ^:const arbitrum-sepolia-chain-id 421614)
+(def ^:const optimism-mainnet-chain-id 10)
+(def ^:const optimism-goerli-chain-id 420)
 (def ^:const optimism-sepolia-chain-id 11155420)
```

Comment:
what about calling the test networks `ethereum-testnet`, `optimism-testnet`, `arbitrum-testnet` ?

Reply:
I would love to. The reason why we renamed it specifically is that we have two test networks (Goerli and Sepolia) at this moment. In future, One will supersede the old, and another will be in the pipeline for testing/integration (It's a cycle).  Additionally, this rename goes well with status-go.